### PR TITLE
Update SceDebugForDriver

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

File: include/psp2kern/kernel/debug.h [hunk 1]
```diff
@@ -129,6 +129,69 @@ int ksceKernelGetMinimumAssertionLevel(void);
...
+
+/**
+ * @brief Get tty log info
+ *
+ * @param[out] buf         - The tty log buffer
+ * @param[in]  buf_size    - The buf size
+ *
+ * @return log length on success, < 0 on error.
+ */
+int ksceKernelGetTtyInfo(char *buf, SceSize buf_size);
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/debug.h:193]: i'm prefer use `TTY` that this is shorten word.
but in unix, we use both of `tty` and `TTY`.
so if another maintainers don't have an opinion of this, i accept this pr.

File: include/psp2kern/kernel/debug.h [hunk 2]
```diff
@@ -129,6 +129,69 @@ int ksceKernelGetMinimumAssertionLevel(void);
...
+} __attribute__((packed)) SceKernelDebugEventLog3;
+
+typedef struct SceKernelDebugEventLog {
+	SceSize size;
+	int data_0x04;    // 0xA, titleid size?
+	char titleid[0xC];
+	int flags;
+	SceUID ppid;
+	SceUID data_0x1C; // thid?
+	int rsvd[4];
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/debug.h:163]: could you expand these variable name to explain it?

File: include/psp2kern/kernel/debug.h [hunk 3]
```diff
@@ -129,6 +129,69 @@ int ksceKernelGetMinimumAssertionLevel(void);
...
+	int data_0x40;    // 0?
+	char ip1[0x10];
+	char ip2[0x10];
+	char ip3[0x10];
+	char ip4[0x10];
+	char ip5[0x10];
+} __attribute__((packed)) SceKernelDebugEventLog3;
+
+typedef struct SceKernelDebugEventLog {
+	SceSize size;      // This struct size(variable size)
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/debug.h:157]: //!<

Comment (d3m3vilurr) [inline include/psp2kern/kernel/debug.h:157]:
```suggestion
	SceSize size;      //!< struct size(variable size)
```
please change inline comment format of struct like this for doxygen.

Comment (Princess-of-Sleeping) [reply include/psp2kern/kernel/debug.h]: done.

**Comment by d3m3vilurr** [comment]
merged 8637b31
I committed with wrong commit message. :(

**Pull request closed without merging**